backend/clubs/models.py

```diff
@@ -636,6 +637,14 @@ class Note(models.Model):
     created_at = models.DateTimeField(auto_now_add=True)
 
 
+class StudentType(models.Model):
+
```

Comment:
A simple docstring here would be nice, to let people know what this is supposed to represent. For example:

```
Represents a student type that the club is intended for. For example, "International Students", "Transfer Students", or "Online Students".
```